go.mod

```diff
@@ -2,36 +2,50 @@ module github.com/prometheus/alertmanager
 
 require (
 	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d
-	github.com/cenkalti/backoff/v4 v4.0.2
+	github.com/armon/go-metrics v0.3.6 // indirect
```

Comment:
Could you remove all the `// indirect` dependencies? I don't think it makes sense to bump them explicitly...

Reply:
Done! Ah they were automatically added, but removing them nuked a lot of vendored code! Nice!